server/src/main/java/org/opensearch/index/translog/RemoteFsTranslog.java

```diff
@@ -241,11 +241,11 @@ private boolean upload(Long primaryTerm, Long generation) throws IOException {
         // primary, the engine is reset to InternalEngine which also initialises the RemoteFsTranslog which in turns
         // downloads all the translogs from remote store and does a flush before the relocation finishes.
         if (primaryModeSupplier.getAsBoolean() == false) {
-            logger.trace("skipped uploading translog for {} {}", primaryTerm, generation);
+            logger.trace("{} skipped uploading translog for {} {}", shardId, primaryTerm, generation);
```

Comment:
shardId can be removed from here. Refer - https://github.com/opensearch-project/OpenSearch/pull/8574/files#r1259279604